src/components/step-sequence/step-sequence.pw.tsx

```diff
@@ -0,0 +1,275 @@
+          "color",
+          color
+        );
+
+        const expectedLabelChild = stepSequenceDataComponentItem(page)
+          .locator("span > span")
+          .nth(0);
+        await expect(expectedLabelChild).toHaveCSS("color", color);
+      });
+    }
+  );
+
+  testData.forEach((hiddenCompleteLabel) => {
+    test(`should check hiddenCompleteLabel set to ${hiddenCompleteLabel}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <StepSequenceItemCustom
+          status="complete"
+          hiddenCompleteLabel={hiddenCompleteLabel}
+        />
+      );
+
+      const expectedLabelChild = stepSequenceDataComponentItem(page)
+        .locator("span")
+        .nth(0);
+      await expect(expectedLabelChild).toHaveText(hiddenCompleteLabel);
+    });
+  });
+
+  testData.forEach((hiddenCurrentLabel) => {
+    test(`should check hiddenCurrentLabel set to ${hiddenCurrentLabel}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <StepSequenceItemCustom
+          status="current"
+          hiddenCurrentLabel={hiddenCurrentLabel}
+        />
+      );
+      const expectedLabelChild = stepSequenceDataComponentItem(page)
+        .locator("span")
+        .nth(0);
+      await expect(expectedLabelChild).toHaveText(hiddenCurrentLabel);
+    });
+  });
+
+  test("should check hideIndicator prop when status is set to complete", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<StepSequenceItemCustom status="complete" hideIndicator />);
+    const expectedLabelChild = stepSequenceDataComponentItem(page).locator(
+      ICON
+    );
+    await expect(expectedLabelChild).toBeVisible();
+    await expect(stepSequenceDataComponentItem(page)).toHaveCSS(
+      "color",
+      "rgb(0, 138, 33)"
+    );
+  });
+
+  ([
+    ["current", "rgba(0, 0, 0, 0.9)"],
+    ["incomplete", "rgba(0, 0, 0, 0.55)"],
+  ] as [StepSequenceItemProps["status"], string][]).forEach(
+    ([status, color]) => {
+      test(`should check hideIndicator prop when status is set to ${status}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<StepSequenceItemCustom status={status} hideIndicator />);
+        const expectedLabelChild = stepSequenceDataComponentItem(page).locator(
+          ICON
+        );
+        await expect(expectedLabelChild).not.toBeVisible();
+        await expect(stepSequenceDataComponentItem(page)).toHaveCSS(
+          "color",
+          color
+        );
+      });
+    }
+  );
+});
+
+test.describe("Accessibility tests for StepSequence component", () => {
+  ["horizontal", "vertical"].forEach((orientation) => {
+    test(`should check orientation set to ${orientation}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<StepSequenceComponent orientation={orientation} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  test("should check children for accessibility tests", async ({
```

Comment:
Or something like that. 

```suggestion
  test("should check children for StepSequenceItemCustom example", async ({
```